timetracker/timetracker/urls.py

```diff
@@ -19,7 +19,20 @@
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+	name = request.GET.get("name", None)
+	if name:
+		res = "Welcome %r" % name
+	else:
+		res = 'No name... please enter one'
+	http_res = '<html><body>'
```

Comment:
We haven't covered it in class but you can use triple quotes (`"""`) to create multi-line strings in Python. There's also the `.format()` function for add variables into your strings.

```
http_res = """<html><body>
    <b>{}</b>
    <form method="get">
        <input type="text" name="name">
        <input type="submit" value="Submit"/>
    </form>
</body></html>""".format(res)
```